### ISSUE TITLE
`update_vsphere_vms` requires uniqueness of IPs accross cluster

### ISSUE THREAD

**Issue by MatthiasWinzeler** [cloudfoundry-incubator/kubo-release#129]
Hi guys

We're deploying kubo to vsphere. We have multiple tenants (with separate vsphere users and folders) where we deploy one kubo cluster each. They all use the same IP range.

We can deploy the first cluster just fine, but on the second one, the `pre_start` script of the `kubelet` on the `worker` fails. This is caused by this [part of the script](https://github.com/cloudfoundry-incubator/kubo-release/blob/master/jobs/kubelet/templates/bin/pre-start.erb#L31-L36):
```
update_vsphere_vms() {
   export GOVC_URL="$(get_cloud_property "server"):$(get_cloud_property "port")"
   export GOVC_USERNAME="$(get_cloud_property "user")"
   export GOVC_PASSWORD="$(get_cloud_property "password")"
   export GOVC_INSECURE="$(get_cloud_property "insecure-flag")"

   /var/vcap/packages/govc/bin/govc vm.change -e 'disk.enableUUID=1' -vm.ip=<%= spec.ip %>
}
```

This snippet updates the disks of the current worker which is needed by the k8s vsphere cloud provider [as stated in their doc.](https://kubernetes.io/docs/getting-started-guides/vsphere/#enable-vsphere-cloud-provider) 

However, the snippet assumes that the current VM can be uniquely identified by its IP. We think this assumption is dangerous, since the VSphere used for the cloud provider requires the `System.View` on datacenter level, so every VSphere tenant user can list all VMs. 

So basically when this script is run on the second cluster, it finds the VM of the first cluster, tries to update it and fails (since it (luckily) lacks the permission to *modify* the VM of the other tenant).

What do you think about changing this behavior? It's probably better to identify the current VM by filtering for the instance uuid (which is stored as custom attribute on the VM; can be retrieved by looping through the VMs, needs some additional `govc` statements probably).
Would you welcome a PR for this?

**Comment by bsnchan** [comment]
Hi @MatthiasWinzeler,

Thanks for catching this. We're definitely open to a PR to improve our release.

**Comment by MatthiasWinzeler** [comment]
Hi @bsnchan 
Looks like there is an elegant way to achieve this using the vm uuid - I filed #141 for it.
